### PR TITLE
Add badges to README.md

### PR DESCRIPTION
**Pull request by JacobRoe**
OPTM-90

### CODE REVIEW

**Review by guisedlythe — CHANGES_REQUESTED** [review]
We should also add the license to this and and the lww_crdt repository via github (if we choose this license and have checked if we are allowed to apply it with the libraries that we use).

File: README.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 # P2P Task Manager
+![CI](https://github.com/PJ-ODS-2021/P2P-Task/actions/workflows/main.yaml/badge.svg)
```
Comment (guisedlythe) [inline README.md]: Please add a newline after the section header:
https://github.com/DavidAnson/markdownlint/blob/v0.23.1/doc/Rules.md#md022

**Review by guisedlythe — CHANGES_REQUESTED** [review]

**Review by firstdayofjune — APPROVED** [review]

**Comment by firstdayofjune** [comment]
Thanks for your changes. I will check the license compatibility and also look for a way to display the used packages in the settings section.

**Review by firstdayofjune — DISMISSED** [review]

**Review by guisedlythe — DISMISSED** [review]

**Comment by firstdayofjune** [comment]
I added the GPL-v3 license to the project, which makes the app truly free software.
I therefore added everyone as a reviewer to confirm the choice of license.
Refer to https://choosealicense.com/licenses/gpl-3.0/ or https://www.gnu.org/licenses/quick-guide-gplv3.html for more info on the GPL.

**Review by firstdayofjune — DISMISSED** [review]

**Review by Tan-jay — DISMISSED** [review]

**Review by guisedlythe — DISMISSED** [review]

**Review by bithianjuman — DISMISSED** [review]

**Review by firstdayofjune — APPROVED** [review]

**Pull request merged**